specs/driver_spec.rb

```diff
@@ -76,5 +81,53 @@
       driver = RideShare::Driver.new(id: 54, name: "Rogers Bartell IV", vin: "1C9EVBRM0YBC564DZ")
       driver.average_rating.must_equal 0
     end
-  end
-end
+  end # average_rating
+
+
+  describe 'total_revenue' do
+    it "calculates take home pay for driver" do
+      trips = [
+        RideShare::Trip.new({cost: 10, rating: 3, start_time: Time.parse('2015-05-20T14:14:00+00:00'),
+          end_time: Time.parse('2015-05-20T14:30:00+00:00')}),
+        RideShare::Trip.new({cost: 4, rating: 3, start_time: Time.parse('2015-05-20T14:14:00+00:00'),
```

Comment:
This is just the sort of thing you could DRY up using a `before` block. If you put it in the outermost `describe` (the one for `Driver`), it will be available inside all the nested `describe`s (for the individual methods).